apps/state/config/config.exs

```diff
@@ -75,6 +75,10 @@ config :state, :shape,
     "831_0008" => 2,
     # Green-C
     "831_0009" => 2,
+    # Green-D (Medford/Tufts)
+    "8000020" => -1,
+    # Green-D (Medford/Tufts)
+    "8000021" => -1,
```

Comment:
I don't know enough about how this config works to evaluate: my guess is this adjusts these shape priority to `-1`. But why do that for these shapes? :O

Reply:
same question, the rest looks good

Reply:
Yeah, so these shapes in particular go Riverside <-> Medford/Tufts and are used for the atypical early-morning scheduled trips. We usually specify these unusual shapes here so that the API does not attempt to add the extra stops to the list of stops belonging to a given route-direction.

FWIW, though, the `stops_on_route` function is a bit of a black box to me and likely needs some retooling so that it doesn't need to be dependent on specifying `shape_id`s (eg, instead of relying only on `route_pattern_id`s).

Reply:
hmm, that sounds less than ideal, but also a problem for another day. makes sense in the meantime!